build-wasm.bat

```diff
@@ -1,56 +1,56 @@
 @echo off
 
-:set the path where your web server's root is (iProgramInCpp's is at C:\gtcache -- don't ask why)
-set WEBSRVROOT=C:\gtcache
+::set the path where your web server's root is (iProgramInCpp's is at C:\gtcache -- don't ask why)
+set WEBSRVROOT=D:\Minecraft\MinecraftWebSite\mcpe
```

Comment:
Change this back

Reply:
Why? It's only ever going to work for one person right off the bat. What difference does it make being your path or my path? It's intended that the executor of the script change it anyways.